app/src/userpages/components/ProductsPage/Header/index.jsx

```diff
@@ -60,7 +60,7 @@ const Header = ({ className, searchComponent, filterComponent }: Props) => {
                             src={product.imageUrl || ''}
                             alt={product.name || ''}
                         />
-                        <NameAndEmail name={product.name} email={product.beneficiaryAddress} />
+                        <NameAndEmail name={product.name} username={product.beneficiaryAddress} />
```

Comment:
Kinda like `<NameAndUsername>` now :)

Reply:
Good point, renamed in https://github.com/streamr-dev/streamr-platform/pull/896/commits/12ffee77027f67ca801afe85db88022ae77ac9f2